addons/account/models/account_move.py

```diff
@@ -3205,7 +3205,7 @@ class AccountMoveLine(models.Model):
     sequence = fields.Integer(default=10)
     name = fields.Char(string='Label', tracking=True)
     quantity = fields.Float(string='Quantity',
-        default=1.0, digits='Product Unit of Measure',
+        default=lambda self: 0 if self._context.get('default_display_type', False) else 1.0, digits='Product Unit of Measure',
```

Comment:
You don't need to specify the default value if you just want something falsy. The default value of `get()` is `None`

Reply:
@william-andre the commit is updated :ok_hand: